### PR TITLE
NIFIREG-387 - Upgraded spring boot and some additional depedencies re…

### PR DESCRIPTION
**Pull request by thenatog**
…quired to allow this upgrade.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -94,10 +94,11 @@
...
         <jetty.version>9.4.19.v20190610</jetty.version>
         <jax.rs.api.version>2.1</jax.rs.api.version>
-        <jersey.version>2.27</jersey.version>
+        <jersey.server.version>2.29.1</jersey.server.version>
+        <jersey.client.version>2.27</jersey.client.version>
         <jackson.version>2.9.9</jackson.version>
         <jackson.databind.version>2.9.10.3</jackson.databind.version>
-        <spring.boot.version>2.1.12.RELEASE</spring.boot.version>
+        <spring.boot.version>2.2.6.RELEASE</spring.boot.version>
         <spring.security.version>5.1.8.RELEASE</spring.security.version>
```
Comment (kevdoran) [inline pom.xml:102]: seems like spring security can be bumped to along with the new version of spring-framework, right?

File: pom.xml [hunk 2]
```diff
@@ -94,10 +94,11 @@
         <org.slf4j.version>1.7.12</org.slf4j.version>
         <jetty.version>9.4.19.v20190610</jetty.version>
         <jax.rs.api.version>2.1</jax.rs.api.version>
-        <jersey.version>2.27</jersey.version>
+        <jersey.server.version>2.29.1</jersey.server.version>
+        <jersey.client.version>2.27</jersey.client.version>
         <jackson.version>2.9.9</jackson.version>
         <jackson.databind.version>2.9.10.3</jackson.databind.version>
```
Comment (kevdoran) [inline pom.xml:100]: looks like jackson can go to 2.10.3. also not sure why we are at separate versions for jackson and jackson databank

File: pom.xml (hunk 1 above)
Comment (thenatog) [reply pom.xml]: Updated to 5.1.9.RELEASE.

File: pom.xml (hunk 2 above)
Comment (thenatog) [reply pom.xml]: Updated to 2.10.3 and removed the jackson.databind.version parameter and changed all dependencies to use only jackson.version.

**Comment by thenatog** [comment]
Update the NOTICE file.

**Comment by alopresto** [comment]
@kevdoran did you need another look at this or were updating the `NOTICE` file and going to `2.10.3` the only reservations you had?

**Comment by kevdoran** [comment]
Hey @alopresto @thenatog - sorry, this one fell off my radar. That was all I had. +1 from me. Thanks!

**Pull request closed without merging**